test/factories/subnet.rb

```diff
@@ -138,7 +138,7 @@
       gateway { '2001:db8:42::1' }
       dns_primary { '2001:db8:42::8' }
       dns_secondary { '2001:db8:42::4' }
-      mtu { '1442' }
+      mtu { '1142' }
```

Comment:
[RFC 2460](https://datatracker.ietf.org/doc/html/rfc2460#section-5) states:
> IPv6 requires that every link in the internet have an MTU of 1280 octets or greater.

Reply:
I haven't introduced this but sure.